wgpu-core/src/track/mod.rs

```diff
@@ -443,9 +441,7 @@ impl<A: HalApi> BindGroupStates<A> {
     /// accesses will be in a constant ascending order.
     pub fn optimize(&mut self) {
         self.buffers.optimize();
-        self.textures.optimize();
         self.views.optimize();
-        self.samplers.optimize();
```

Comment:
Could we leave a note on all the places where we're omitting certain types of resources to know it was intentional and why it's valid